### PR TITLE
Send data from edit form to spreadsheet

### LINKED ISSUE #72: Handle errors from OAuth
When OAuth expires (?), a 500 NullPtrException is thrown. We should be able to prevent requests from being sent until OAuth is set up, and handle errors when the request is sent without OAuth
image

### PR DESCRIPTION
**Pull request by cadecrandall**
### Description
This PR finishes TODOs from #61. The webpage will only accept edits from vendor/account combos that already exist, and then pushes the changes to the spreadsheet. A NullPointerException previously was thrown if OAuth had not been set up, and this PR catches that problem and gives a more descriptive error

### Issues Resolved
- Closes #72

### CODE REVIEW

File: src/main/java/data/JsonConverter.java [hunk 1]
```diff
@@ -35,12 +35,21 @@ public JsonConverter(String filePathName) {
     basePath = filePathName;
   }
 
+  /**
+   * Checks to see if a vendor's configuration already exists, and updates
+   * it if it does
```
Comment (cxfloeder) [inline src/main/java/data/JsonConverter.java:40]: Add period.

Comment (ianjjohnson) [reply src/main/java/data/JsonConverter.java]: +1 to this comment

Comment (ianjjohnson) [reply src/main/java/data/JsonConverter.java]: Also, describe what it does if the config doesn't exist

File: src/main/java/servlets/BillingConfig.java [hunk 2]
```diff
@@ -92,14 +65,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
       JsonConverter jsonConverter = new JsonConverter();
       if (jsonConverter.updateFile(newVendor)) {
+        // Update Google sheets following update file.
+        updateSheets(request);
         // Redirect only when the operation succeeded
         response.getWriter().println(newVendor.getVendorID());
         response.sendRedirect(REDIRECT_READFILE);
       } else {
-        response.sendError(400, "Something went wrong when we tried to write your file.");
+        response.sendError(400, "This configuration does not exist.");
```
Comment (ianjjohnson) [inline src/main/java/servlets/BillingConfig.java:70]: nice

File: src/main/java/servlets/VendorServlet.java [hunk 3]
```diff
@@ -57,7 +57,7 @@ public void doDelete(HttpServletRequest request, HttpServletResponse response)
     throws IOException {  
     String vendorID = request.getParameter(VENDOR_ID_PARAM);
     String responseString = String.format(
-      "Config with VendorID:%s was successfuly deleted.", vendorID); 
+      "Config with VendorID:%s was successfully deleted.", vendorID);
```
Comment (jakecarlson1) [inline src/main/java/servlets/VendorServlet.java:60]: Nit: add a space "VendorID: %s".

**Pull request merged**